### ISSUE TITLE
modal.com

### ISSUE THREAD

**Issue by lord-dubious** [feliks912/ComfyUI_modal#1]
is this meant to be used in modal.com

**Comment by feliks912** [comment]
Yes and is now deprecated due to Modal volumes automatically committing every 10 seconds or so. Until now a manual volume commit had to be called, which there was no way of doing other than calling it in the same shell session. The button added in the lower right corner is made to commit the volume before calling the process to end.

Using modal.com for this purpose is dumb and I've passed onto Vast.ai
